docs/articles/documentation/test-api/actions/action-options.md

```diff
@@ -0,0 +1,85 @@
+* [Click Action Options](#click-action-options)
+* [Typing Action Options](#typing-action-options)
+
+## Mouse Action Options
+
+Provide additional parameters for a mouse action.
+
+```js
+{
+    ctrl: false | true,
+    alt: false | true,
+    shift: false | true,
+    meta: false | true,
+    offsetX: Number,
+    offsetY: Number
+}
+```
+
+Parameter                      | Type    | Description                                                                                                                               | Default
+------------------------------ | ------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------
+`ctrl`, `alt`, `shift`, `meta` | Boolean | Indicate which modifier keys are to be pressed during the mouse action.                                                                   | `false`
+`offsetX`, `offsetY`           | Number  | Mouse pointer coordinates relative to the top-left corner of the target element. Define a point where the action is performed or started. | The center of the target element.
+
+Mouse action options are used in the [t.drag](drag-element.md#drag-an-element-by-an-offset),
+[t.dragToElement](drag-element.md#drag-an-element-onto-another-one) and [t.hover](hover.md) actions.
+
+## Click Action Options
+
+Provide additional parameters for a click action.
+
+```js
+{
+    ctrl: false | true,
+    alt: false | true,
+    shift: false | true,
+    meta: false | true,
+    offsetX: Number,
+    offsetY: Number,
+    caretPos: Number
+}
+```
+
+Parameter                      | Type    | Description                                                                                                                               | Default
+------------------------------ | ------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------
+`ctrl`, `alt`, `shift`, `meta` | Boolean | Indicate which modifier keys are to be pressed during the mouse action.                                                                   | `false`
+`offsetX`, `offsetY`           | Number  | Mouse pointer coordinates relative to the top-left corner of the target element. Define a point where the action is performed or started. | The center of the target element.
+`caretPos`                     | Number  | The initial caret position if the action is performed on a text input field.                                                              | The length of the input field content.
+
+Click action options are used in the [t.click](click.md), [t.doubleClick](double-click.md) and [t.rightClick](right-click.md) actions.
+
+## Typing Action Options
+
+Provide additional parameters for a typing operation.
+
+```js
+{
+    replace: false | true,
```

Comment:
Before "new" option `caretPos` added in the end of list https://github.com/DevExpress/testcafe/pull/683/files#diff-8e662184507eb3363796c47b173723e2R49 But here "new" option `replace` added to the beggining. For more readability let's do it consistent.